build.gradle

```diff
@@ -343,17 +381,34 @@ for (publishedProject in publishedProjects) {
             maven(MavenPublication) {
                 from project.components.java
                 pom {
+                    groupId = project.group
+                    artifactId = project.name
+                    version = project.version
+                    packaging = "jar"
+
+                    name = project.name
                     description = publishedProject.description
+                    url = projectURL
+
                     licenses {
                         license {
                             name = 'The Apache License, Version 2.0'
                             url = 'https://www.apache.org/licenses/LICENSE-2.0.txt'
                             distribution = 'repo'
                         }
                     }
-                    organization {
-                        name = 'R3 LLC'
-                        url = "https://www.r3.com"
+                    developers {
+                        developer {
+                            name = 'Conclave Team'
+                            email = "conclave@r3.com"
+                            organization = 'R3 LLC'
+                            organizationUrl = "https://www.r3.com"
+                        }
+                    }
+                    scm {
+                        connection = "scm:git:git://github.com/R3Conclave/conclave-core-sdk.git"
+                        developerConnection = "scm:git:ssh://github.com:R3Conclave/conclave-core-sdk.git"
```

Comment:
According to the POM docs, this is only for write access, whatever that means. I think we should remove this one.

Reply:
[developers](https://central.sonatype.org/publish/requirements/#scm-information)
I understand they are compulsory, but we can put whatever we like there. But if the users do not have access, it does not seem to be a problem